docs/user/manual.adoc

```diff
@@ -589,6 +589,24 @@ For example, mutable bindings are underlined by default and you can override thi
 }
 ----
 
+Most themes don't have support for styling unsafe operations differently. You can fix this by specifying custom styles in your `settings.json`:
+
+[source,jsonc]
+----
+"editor.semanticTokenColorCustomizations": {
+        "[Theme Name]": {
+            "rules": {
+                "operator.unsafe": "#ff6600",
+                "function.unsafe": "#ff6600"
+                "method.unsafe": "#ff6600"
+            }
+        }
+    },
+}
+----
+
+Make sure you include the brackets around the theme name. For example, use "[Ayu Dark]" to customize the theme Ayu Dark.
+
```

Comment:
It feels like the above two bits of advice duplicate each other a bit. Let's merge that into one code sample and explain the difference between top-level `rules` and `[Theme Name]` syntax.